frontend/src/Movie/History/MovieHistoryTableContent.js

```diff
@@ -72,12 +72,12 @@ class MovieHistoryTableContent extends Component {
 
         {
           !isFetching && !!error &&
-            <div className={styles.blankpad}>Unable to load history</div>
+            <div className={styles.blankpad}>{translate('UnableToLoadHistory')}</div>
```

Comment:
drop lines

Reply:
lint really needs to be updated to get these -_-